### ISSUE TITLE
Display game server logs in cw dashboard

### ISSUE THREAD

**Issue by j3fffff** [StrangeLoopGames/EcoIssues#22249]
@gzatravkin can you advice on how to get server logs from game server api? or is it not possible yet?

**Comment by gzatravkin** [comment]
@j3fffff not, from server api you can't. But there are log files at Eco\Server\Logs if you have disk access

**Comment by denysaw** [comment]
@gzatravkin Nah, it's not ElectionsUI, it's Accounts. We have no direct game server's disk access. Only via API. Could you create an issue for adding some static files sharing endpoint? So we could just request those log files directly by some url? Thanx ;)
